fineract-provider/src/main/java/org/apache/fineract/portfolio/account/api/AccountTransfersApiResourceSwagger.java

```diff
@@ -529,17 +529,17 @@ public static final class PostAccountTransfersRefundByTransferRequest {
         private PostAccountTransfersRefundByTransferRequest() {}
 
         @Schema(example = "2")
-        public Integer fromAccountId;
+        public Long fromAccountId;
         @Schema(example = "1")
-        public Integer fromAccountType;
```

Comment:
This looks wrong to me based on AccountTransfersWritePlatformServiceImpl line 105. It is the ordinal of an enum.

Reply:
Done.